hazelcast/src/test/java/com/hazelcast/internal/networking/nio/iobalancer/MonkeyMigrationStrategyTest.java

```diff
@@ -52,16 +51,16 @@ public class MonkeyMigrationStrategyTest extends HazelcastTestSupport {
 
     @Test
     public void imbalanceDetected_shouldReturnFalseWhenNoPipelineExist() {
-        ownerPipelines.put(imbalance.srcOwner, Collections.<MigratablePipeline>emptySet());
+        ownerPipelines.put(imbalance.srcOwner, Collections.emptySet());
```

Comment:
Minor
Why not `Set.of()` either here?

Reply:
Hi. I just removed the type of the diamond operator. I did not want to change the original test.